test/utils/IexecWrapper.ts

```diff
@@ -98,12 +107,55 @@ export class IexecWrapper {
         return ((workerpoolPrice * stakeRatio) / 100) * volume;
     }
 
+    /**
+     * Compute the amount of RLC tokens to be staked by the
+     * worker in order to contribute to a task.
+     * @param workerpoolAddress address of the workerpool
+     * @param workerpoolPrice price of the workerpool
+     * @returns value of worker stake
+     */
+    async computeWorkerTaskStake(workerpoolAddress: string, workerpoolPrice: number) {
+        // TODO make "m_workerStakeRatioPolicy()" as view function in IWorkerpool.v8 and use it.
```

Comment:
To understand, do you have any issue using `import { IWorkerpool__factory } from '../typechain'` artifact generated from  `IWorkerpool.v8.sol`?

Reply:
Yes, the function `m_workerStakeRatioPolicy()` is not marked as `view` in the file `IWorkerpool.v8.sol` so typechain generator considers it a write transaction and returns a `ContractInteraction` object instead of a number.

* [function m_workerStakeRatioPolicy() external returns (uint256);](https://github.com/iExecBlockchainComputing/PoCo/blob/develop/contracts/registries/workerpools/IWorkerpool.v8.sol#L9)

![image](https://github.com/user-attachments/assets/93200c28-75a4-4744-8940-bed795944918)

Reply:
Okai thanks!